ChaosMod/Effects/db/Peds/PedsJesusArmy.cpp

```diff
@@ -0,0 +1,47 @@
+#include <stdafx.h>
+
+static void OnStart()
+{
+	static constexpr Hash modelHash = -835930287;
+
+	Ped playerPed = PLAYER_PED_ID();
+	Vector3 playerPos = GET_ENTITY_COORDS(playerPed, false);
+	static const Hash playerGroup = GET_HASH_KEY("PLAYER");
+	static const Hash civGroup = GET_HASH_KEY("CIVMALE");
+	static const Hash femCivGroup = GET_HASH_KEY("CIVFEMALE");
+
+	Hash relationshipGroup;
+	ADD_RELATIONSHIP_GROUP("_HOSTILE_JESUS", &relationshipGroup);
+	SET_RELATIONSHIP_BETWEEN_GROUPS(5, relationshipGroup, playerGroup);
+	SET_RELATIONSHIP_BETWEEN_GROUPS(0, relationshipGroup, civGroup);
+	SET_RELATIONSHIP_BETWEEN_GROUPS(0, relationshipGroup, femCivGroup);
+
+	for(int i = 0; i < 50; i++)
```

Comment:
I would suggest not spawning 50 jesus’ as it could cause game crashes limit it to 10 or 15.

Reply:
fixed